src/v2/modules/calculateBearing.mjs

```diff
@@ -0,0 +1,39 @@
+// -----------------------------------------------------------------------------
+// Copyright 2022 Open Climate Tech Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+/**
+ * Determines the relative direction of `end` from `start`.
+ *
+ * @param {Array} start - Origin latitude-longitude pair.
+ * @param {Array} end - Destination latitude-longitude pair.
+ *
+ * @returns {number} The relative direction of `end` from `start` in degrees.
+ */
+export default function calculateBearing(start, end) {
```

Comment:
It probably makes the most sense to plumb this stuff through (or, less likely, share implementation between the data side and the app server/client)

Reply:
Using `fireHeading`. Note that `fireHeading` (because it’s a DB thing?) may not be defined even though the bearing can be calculated from the camera’s primary POV polygon. As a result, when `fireHeading` isn’t defined, markers point due north even though their polygons render otherwise.